core/templates/dev/head/pages/library/Library.js

```diff
@@ -80,6 +80,36 @@ oppia.controller('Library', [
       $http.get('/libraryindexhandler').success(function(data) {
         $scope.libraryGroups = data.activity_summary_dicts_by_category;
 
+        $scope.userIsLoggedIn = GLOBALS.userIsLoggedIn;
+        var activityDetails = {};
+        if ($scope.userIsLoggedIn) {
+          $scope.activitiesOwned = {explorations: {}, collections: {}};
+          $http.get('/creatordashboardhandler/data')
+            .then(function(response) {
+              $scope.libraryGroups.forEach(function(activityGroups) {
+                var activity = activityGroups.activity_summary_dicts;
+
+                activity.forEach(function(activity) {
```

Comment:
(Should be handled by the comment above, but note that it's bad practice to have the name of the array you're iterating over, and the name of an individual element of that array, be the same thing.)